### ISSUE TITLE
BufferUnderflowException with leaves quality fancy and enabled cull leaves

### ISSUE THREAD

**Issue by eszdman** [TeamMidnightDust/CullLeaves#37]
Hardware:
- Ryzen 5600
- 24GB RAM (allocated 8gb 30% used)
- RX580 8GB

Crash:
```
java.nio.BufferUnderflowException: null
	at me.jellysquid.mods.sodium.client.gl.util.MultiDrawBatch$UnsafeMultiDrawBatch.add(MultiDrawBatch.java:155) ~[sodium-fabric-mc1.19.2-0.4.4+build.18.jar:?]
	at me.jellysquid.mods.sodium.client.render.chunk.RegionChunkRenderer.addDrawCall(RegionChunkRenderer.java:190) ~[sodium-fabric-mc1.19.2-0.4.4+build.18.jar:?]
	at me.jellysquid.mods.sodium.client.render.chunk.RegionChunkRenderer.redirect$cam000$cullleaves$fixTransparentBlockFaceCulling(RegionChunkRenderer.java:2063) ~[sodium-fabric-mc1.19.2-0.4.4+build.18.jar:?]
```

Versions:
- Sodium 1.19.2-0.4.4+build.18
- Minecraft 1.19.2
- CullLeaves latest 3.0.1

Full log if necessary:
[latest.log](https://github.com/TeamMidnightDust/CullLeaves/files/10950700/latest.log)

**Comment by Commander07** [comment]
I'm not sure if you are still having this issue but on version 3.0.3 changing the config to the below fixed it for me
```json
{
  "enabled": true,
  "sodiumBlockFaceCullingFix": false
}
```

**Comment by xiPakuL** [comment]
This did not fix it for me and i am experiencing this same issue on 1.19.X Forge

**Comment by xilenasu** [comment]
@Commander07's solution solved it for me in version 3.0.1, mc 1.19.2
